src/build/__main__.py

```diff
@@ -105,6 +108,12 @@ def main_parser():  # type: () -> argparse.ArgumentParser
         default=cwd,
         help='source directory (defaults to current directory)',
     )
+    parser.add_argument(
+        '--version',
+        '-V',
+        action='version',
+        version='build {} ({})'.format(build.__version__, ', '.join(path for path in paths if path)),
```

Comment:
With regards to the `__path__` vs `__file__` thing, I think either is essentially correct, but Filipe's version is I guess slightly more correct?

I'm not sure I'd be able to understand the information I'm being shown if there were multiple paths, but I think I'd probably rather see that something funky is going on than not?